Files/ViewModels/SettingsViewModel.cs

```diff
@@ -50,6 +53,7 @@ private async Task<SettingsViewModel> Initialize()
             FileTagsSettings = new FileTagsSettings();
 
             // Send analytics to AppCenter
+            StartAppCenter();
```

Comment:
If the purpose of this change is to make sure that `AppCenter.Start()` is called before `TrackAnalytics()` note that the `async void` method will return before reaching it.

Reply:
Would `await` solve this issue?

Reply:
Yup